### DISCUSSION TITLE
1.10

### DISCUSSION THREAD

**Discussion by kirb** [LegacyUpdate/LegacyUpdate#266] [Announcements]
> [!IMPORTANT]
> If nothing happens when you open Legacy Update setup on Windows XP, [follow these steps](https://legacyupdate.net/faq/issues) to work around a Windows bug.

Happy 10th release! 🎉

Legacy Update 1.10 comes packed with numerous improvements to celebrate our 10th major version.

* **Setup is faster than ever:** Many improvements have been made to the initial Legacy Update setup tool.
  * If a restart is required to install Service Packs, Servicing Stacks, and other updates, your system will enter “setup mode”. You’ll see Legacy Update reappear immediately after the Windows splash screen. This makes the process go by faster, and fixes issues with the previous method we used, such as the computer automatically locking itself while logged into the LegacyUpdateTemp user.
  * Installing updates on Windows Vista and later better avoid Windows Update’s high CPU usage issues.
  * Installation of Windows XP Service Pack 3 on top of RTM is now faster, using the smaller SP1a rather than SP2, and avoiding an additional restart.
* **Windows 2000:**
  * Internet Explorer 6 setup now downloads all of its required files before restarting your computer. This can help when internet isn’t connected immediately on startup (e.g. with Wi-Fi or dial-up).
  * Possibly fixes “No mapping between account names and security IDs was done.” error when running Legacy Update on the first boot after installing Service Pack 4.
* **Windows 7:**
  * Fixes some required updates being skipped on Windows 7 installations that have already been partially updated.
  * Fixes a required update failing to install on Windows 7.
* **Windows 8:**
  * As Microsoft removed the Windows 8.1 download from their website in June 2024, the feature to upgrade Windows 8 to 8.1 has been removed. You’ll need to perform this upgrade manually.
* **Windows 10 and 11:**
  * When clicking the Restart button on the website, Legacy Update now ensures the system is aware that you’re requesting to “update and restart”. Previously, this may have triggered a regular restart without installing updates.
  * Windows 10 changes the way Windows Update logs work, requiring a PowerShell cmdlet to be executed to convert event logs to a text file. When clicking “View Log File” on the website, the log will be automatically generated and opened for you.
* **New:**
  * Adds a nicer Windows Aero-style look in Legacy Update setup under Windows Vista and later.
  * Legacy Update setup now looks for unsupported configurations, such as running on a beta release of Windows, or using Compatibility Mode on the setup executable. Updates tend to not work correctly in these cases, and could damage your Windows installation.
  * Legacy Update now has a “launcher” executable. This should bring no noticeable difference for the majority of users, but it may be more convenient to those using custom shortcuts/scripts to launch Legacy Update. The `rundll32` method is now deprecated, but will continue to work.
  * Adds a `/passive` flag to the setup, skipping all user input and automatically installing with the default options.
* **Fixes:**
  * Fixes setup failing to run on 486, Pentium, and Cyrix systems (CPUs lacking SSE instructions).
  * Fixes Windows Update’s installation progress dialog appearing blurry on systems using display scaling, and appearing behind the Internet Explorer window.
  * General cleanup, some file size reduction, and other small improvements.

A big thanks to @renodr for his awesome contributions to this release, including fixing some bugs, and helping to set up continuous integration (CI) tests on GitHub Actions, which allows us to provide nightly builds.

Thanks for using Legacy Update! I’m currently looking at acquiring Windows Phone, Surface, and maybe even Windows Mixed Reality hardware to review what Legacy Update can do to improve these discontinued products, so your [support](https://legacyupdate.net/#sponsors) is always appreciated.

<hr /><em>This discussion was created from the release <a href='https://github.com/LegacyUpdate/LegacyUpdate/releases/tag/'>1.10</a>.</em>

**Comment by 44cK3r** [comment]
Oh man I'm excited for compatibility with windows phones I just hope it's CE Based ones you are referring to the 8/8.1/10 Based ones would be nice as well as I do have a couple but the whole lot plus palm Based ones and others I don't know of would be absolutely AMAZING man I love Legacy Update

**Reply by kirb** [reply]
It would be WP8 - 10, because they use the same Windows Update infrastructure as desktop Windows, including other things built on WU like the Windows Store. It's of a bit more interest currently, because some services for them went offline only this month. But once some of that stuff is up and running, WinCE is always a possibility 😉

**Reply by Torinde** [reply]
Would you support upgrading Windows Phone to WinRT/10 (via the various unofficial methods)?
